ci/build.ps1

```diff
@@ -50,6 +50,9 @@ if ($compiler -eq 'MINGW') {
   # Add MinGW to the PATH
   $env:PATH = "C:\msys64\mingw$bits\bin;$env:PATH"
 
+  # Avoid pacman "warning" which causes non-zero return code. https://github.com/open62541/open62541/issues/2068
+  & C:\msys64\usr\bin\mkdir -p /var/cache/pacman/pkg
```

Comment:
cc @janlazo

Reply:
Is this suppose to create `C:\msys64\var\cache\pacman\pkg`?